spec/benchmark-spec.js

```diff
@@ -0,0 +1,21 @@
+/** @babel */
+/* eslint-env jasmine, atomtest */
+
+/* This file contains all specs to test the startup/load-performance indicated by THE timecop. */
+
+describe('editorconfig', () => {
+	beforeEach(() => {
+		waitsForPromise(() => atom.packages.activatePackage('editorconfig'));
+	});
+
+	describe('when activated', () => {
+		const loadLimit = 50; // sets the limit for package-loading in ms.
+		it(`should being loaded in less than ${loadLimit}ms`, () => {
+			expect(atom.packages.getLoadedPackage('editorconfig').loadTime).toBeLessThan(loadLimit);
```

Comment:
Not entirely sure how objective this test is. It is very subjective to the machine it is being run on. And for automated tests such as Travis, I don't believe there is a guarantee to the type of machine it is being run on causing it to sometimes fail and sometimes pass.